bin/copy_dependencies_vcpkg.bat

```diff
@@ -0,0 +1,15 @@
+ECHO OFF
```

Comment:
My batch file skills are pretty rusty, but I have a feeling that this should maybe be ``@ECHO OFF`` so that the command itself isn't written to the terminal. Is that right...?

Reply:
In this context (for whatever reason) that doesn't happen but I think you're correct. Batch files are the worst - especially all the hacks they've added over time (if you ever want to be horrified try and understand what ``SETLOCAL ENABLEDELAYEDEXPANSION`` does)